ui/conductor/src/traits/electricDemand/electric.js

```diff
@@ -62,15 +62,15 @@ export function useElectricDemand(value) {
   const _v = computed(() => toValue(value));
 
   // return (v / d) or 0 if v is undefined
-  const div = (v, d) => v ? v / d : 0;
+  const div = (v, d) => v ? v / d : null;
```

Comment:
Not quite correct. 
1. The comment no longer matches the code
2. Returns null if v is 0, not just if v is null or undefined